### ISSUE TITLE
Nuget installation fails

### ISSUE THREAD

**Issue by pezholio** [alphagov/notifications-net-client#55]
Following the instructions to install with Nuget, I get a 404:

```
nuget install Notify -Source https://api.bintray.com/nuget/gov-uk-notify/notifications-net-client
Feeds used:
  https://api.bintray.com/nuget/gov-uk-notify/notifications-net-client

Installing package 'Notify' to 'C:\Users\fabio'.
  GET https://api.bintray.com/nuget/gov-uk-notify/notifications-net-client/FindPackagesById()?id='Notify'&semVerLevel=2.0.0
  NotFound https://api.bintray.com/nuget/gov-uk-notify/notifications-net-client/FindPackagesById()?id='Notify'&semVerLevel=2.0.0 124ms
An error occurred while retrieving package metadata for 'Notify' from source 'https://api.bintray.com/nuget/gov-uk-notify/notifications-net-client'.
  Failed to fetch results from V2 feed at 'https://api.bintray.com/nuget/gov-uk-notify/notifications-net-client/FindPackagesById()?id='Notify'&semVerLevel=2.0.0' with following message : Response status code does not indicate success: 404 (Not Found).
  Response status code does not indicate success: 404 (Not Found).
```

I also tried `nuget install Notify -Source https://api.bintray.com/gov-uk-notify/nuget/Notify` guessing based on the structure of the package URL and I got prompted for a username and password.

**Comment by allait** [comment]
Hi @pezholio, thank you for reporting this! Looks like the instructions in our README are outdated, could you let me know if the following works for you:

```
nuget sources Add -Name NotifyBintray -Source https://api.bintray.com/nuget/gov-uk-notify/nuget
nuget install Notify
```

**Comment by pezholio** [comment]
Cool, works for me. I'll submit a PR

**Comment by allait** [comment]
Fixed by #56
